### PR TITLE
Fix unauthorized_client error on Login for Visual Studio F5 experience

### PR DESCRIPTION
**Pull request by fcastells**
By setting the correct DockerServiceUrl in docker-compose.dcproj

### CODE REVIEW

File: src/docker-compose.dcproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>fea0c318-ffed-4d39-8781-265718ca43dd</ProjectGuid>
     <DockerLaunchBrowser>True</DockerLaunchBrowser>
-    <DockerServiceUrl>http://localhost:5100</DockerServiceUrl>
+    <DockerServiceUrl>http://docker.for.win.localhost:5100</DockerServiceUrl>
```
Comment (mvelosop) [inline src/docker-compose.dcproj:6]: The address should be `host.docker.internal`, as docker.for.win.localhost is now deprecated.
```suggestion
    <DockerServiceUrl>http://host.docker.internal:5100</DockerServiceUrl>
```

**Review by mvelosop — CHANGES_REQUESTED** [review]
Hi @fcastells, thanks for the PR 😊

Just a minor tweak and it'd be good to go!

**Comment by sughosneo** [comment]
> By setting the correct DockerServiceUrl in docker-compose.dcproj

Hello @fcastells , thank you for submitting this PR. 
As @mvelosop, suggested in his review, Could you please make respective changes and update the PR ?

**Comment by fcastells** [comment]
@sughosneo @mvelosop sorry for never following up on this. I forgot the exact details, but I think the requested change is not that simple, because that URL was also somewhere else (In a test database, I believe) and I wanted to make sure of it. But I never did... I'll look at it and submit it to review again.

**Comment by sughosneo** [comment]
Hi @fcastells , the required change has already been taken care part of `.NET 5` update. So I am closing this PR as of now. 
You can take the latest from the `dev` branch and test out the changes at your end. 

Thank you.

**Pull request closed without merging**